cli/flags.rs

```diff
@@ -324,19 +326,13 @@ lazy_static! {
 }
 
 /// Main entry point for parsing deno's command line flags.
-/// Exits the process on error.
-pub fn flags_from_vec(args: Vec<String>) -> Flags {
-  match flags_from_vec_safe(args) {
-    Ok(flags) => flags,
-    Err(err) => err.exit(),
-  }
-}
-
-/// Same as flags_from_vec but does not exit on error.
-pub fn flags_from_vec_safe(args: Vec<String>) -> clap::Result<Flags> {
```

Comment:
What's the reason for removing this function?

Reply:
The exiting one is removed because I wanted to unify that with the other errors in `main.rs`, under `unwrap_or_exit()`.

Reply:
I see, given number of lines changed this change caused I don't think it's worth it and using validator (https://github.com/denoland/deno/pull/7369#discussion_r552255305) would help as well

Reply:
No the line changes are from renaming `flags_from_vec_safe()` to the now available `flags_from_vec()`. I think it's better to only have one of these either way.